public/Aryan Suri/Sidebar/sidebar.js

```diff
@@ -594,871 +653,773 @@ function createSidebar() {
+    }
+  });
+
+
+  window.addEventListener('click', function (event) {
+    if (event.target == document.getElementById("refer_usage")) {
+      document.getElementById("sidebar4").style.width = "480px";
+      document.getElementById("sidebar2").style.width = "0px";
+
+      //document.getElementById("main").style.marginLeft = "250px";
+    }
+  });
+
+  window.addEventListener('click', function (event) {
+    if (event.target == document.getElementById("developer_usage")) {
+      document.getElementById("sidebar4").style.width = "480px";
+      document.getElementById("sidebar3").style.width = "0px";
+
+      //document.getElementById("main").style.marginLeft = "250px";
+    }
+  });
+
+  $(".mt-container-secondary").css("display", "none");
+
+  $('#per_table').click(function () {
+    if ($("#iFrameResizer0").is(":hidden")) {
+      $("#iFrameResizer0").slideDown("slow");
+
+    } else {
+      $("#iFrameResizer0").slideUp("slow");
+
+    }
+  });
+
+  $('#gloss_table').click(function () {
+    if ($("#gloss_table_put").is(":hidden")) {
+
+      $("#gloss_table_put").load("https://chem.libretexts.org/Bookshelves/Ancillary_Materials/Reference/Organic_Chemistry_Glossary #a9abfa27-98d2-c6fa-0c61-a753f2b8dcb1");
+      $("#gloss_table_put").slideDown("slow");
+    } else {
+      $("#gloss_table_put").slideUp("slow");
+    }
+  });
+
+  $('#ref_table').click(function () {
+    if ($("#ref_table_put").is(":hidden")) {
+
+      $("#ref_table_put").slideDown("slow");
+    } else {
+      $("#ref_table_put").slideUp("slow");
+    }
+  });
+
+  $('#phy_table').click(function () {
+    if ($("#phy_table_put").is(":hidden")) {
+
+      $("#phy_table_put").load("https://chem.libretexts.org/Bookshelves/Ancillary_Materials/Reference/Units_and_Conversions/Physical_Constants #pageText");
+      $("#phy_table_put").slideDown("slow");
+    } else {
+      $("#phy_table_put").slideUp("slow");
+    }
+  });
+
+
+  $('#conv_table').click(function () {
+    if ($("#conv_table_put").is(":hidden")) {
+
+      $("#conv_table_put").load("https://chem.libretexts.org/Bookshelves/Ancillary_Materials/Reference/Units_and_Conversions #pageText");
+      $("#conv_table_put").slideDown("slow");
+    } else {
+      $("#conv_table_put").slideUp("slow");
+    }
+  });
+
+  $('#conversion_table').click(function () {
+    if ($("#conversion_table_put").is(":hidden")) {
+
+      $("#conversion_table_put").slideDown("slow");
+    } else {
+      $("#conversion_table_put").slideUp("slow");
+    }
+  });
+
+  $('#construction-guide').click(function () {
+    if ($("#construction-guide-put").is(":hidden")) {
+
+      $("#construction-guide-put").slideDown("slow");
+    } else {
+      $("#construction-guide-put").slideUp("slow");
+    }
+  });
+
+  $('#library-guide').click(function () {
+    if ($("#library-guide-put").is(":hidden")) {
+
+      $("#library-guide-put").slideDown("slow");
+    } else {
+      $("#library-guide-put").slideUp("slow");
+    }
+  });
```

Comment:
This bit of code also seems easily consolidatable. I would recommend deciding whether to use dashes or underscores in these id names for consistency.